spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -591,12 +1062,19 @@ def expect_resubmit_page_inline_selfie_error_message(should_be_present)
     end
   end
 
+  def continue_to_selfie_upload
+    if IdentityConfig.store.doc_auth_separate_pages_enabled
+      click_continue
+    end
+  end
+
```

Comment:
```suggestion
```

can we remove this method? .. it's used in 1 place only and simply calling click_continue in the test is easier to read